docs/source/examples/cirq-ibmq-backends.myst

```diff
@@ -0,0 +1,255 @@
+print(cirq_circuit)
+```
+
+
+We will use the probability of the ground state as our observable to mitigate, the expectation value of which should
+evaluate to one in the noiseless setting.
+
+
+## High-level usage
+
+
+To use Mitiq with just a few lines of code, we need to define an executor, a function which inputs a circuit and outputs
+the expectation value to mitigate. This function will:
+
+1. [Optionally] Add measurement(s) to the circuit.
+2. Run the circuit.
+3. Convert from raw measurement statistics (or a different output format) to an expectation value.
+
+For information on how to define more advanced executors, see the {ref}`Executors <guide/executors/executors>` section of the Mitiq user guide.
+
+We define the executor function in the following code block. Because we are using IBMQ backends, we first load our account.
+
++++
+
+**Note:** Using an IBM quantum computer requires a valid IBMQ account. See <https://quantum-computing.ibm.com/>
+for instructions to create an account, save credentials, and see online quantum computers.
+
+```{code-cell} ipython3
+import qiskit
+from mitiq.interface.mitiq_qiskit.conversions import to_qiskit
+
+if qiskit.IBMQ.stored_account() and USE_REAL_HARDWARE:
+    provider = qiskit.IBMQ.load_account()
+    backend = provider.get_backend("ibmq_qasm_simulator")  # Set quantum computer here!
+else:
+    # Default to a simulator.
+    backend = qiskit.Aer.get_backend("qasm_simulator"),
+
+
+def cirq_ibm_executor(cirq_circuit: cirq.Circuit, shots: int = 1024) -> float:
+    """Returns the expectation value to be mitigated.
+
+    Args:
+        circuit: Circuit to run.
+        shots: Number of times to execute the circuit to compute the expectation value.
+    """
+    # convert from Cirq circuit to Qiskit circuit.
+    circuit = to_qiskit(cirq_circuit) 
+    
+    if USE_REAL_HARDWARE:
+        # Run the circuit on hardware
+        job = qiskit.execute(
+            experiments=circuit,
+            backend=backend,
+            optimization_level=0,  # Important to preserve folded gates.
+            shots=shots
+        )
+    else:
+        # Simulate the circuit with noise
+        noise_model = initialized_depolarizing_noise(noise_level=0.02)
+        job = qiskit.execute(
+            experiments=circuit,
+            backend=qiskit.Aer.get_backend("qasm_simulator"),
+            noise_model=noise_model,
+            basis_gates=noise_model.basis_gates,
+            optimization_level=0,  # Important to preserve folded gates.
+            shots=shots,
+        )
+
+    # Convert from raw measurement counts to the expectation value
+    counts = job.result().get_counts()
+    if counts.get("0") is None:
+        expectation_value = 0.
+    else:
+        expectation_value = counts.get("0") / shots
+    return expectation_value
+```
+
+
+At this point, the circuit can be executed to return a mitigated expectation value by running ``zne.execute_with_zne``,
+as follows.
```

Comment:
```suggestion
At this point, the circuit can be executed to return a mitigated expectation value by running Mitiq's [`zne.execute_with_zne`](https://mitiq.readthedocs.io/en/stable/apidoc.html?highlight=execute_with_zne#mitiq.zne.zne.execute_with_zne) function as follows.
```

Is this the right syntax for a link to this page?

Reply:
FYI: 
The right syntax to link to a Mitiq function is this:
https://github.com/unitaryfund/mitiq/blob/f06221553cd73703bb5a9a815b88682ad705b13b/docs/source/examples/zne-braket-ionq.myst#L122

The right syntax to link to an internal page of the docs is this: https://github.com/unitaryfund/mitiq/blob/f06221553cd73703bb5a9a815b88682ad705b13b/docs/source/examples/zne-braket-ionq.myst#L21

Adding URLs is dangerous since the version of the docs ("stable" in this case) is hard-coded and they tend to get broken when the docs is modified.

Reply:
This link will render strangely in the Jupyter notebook but it’s better to avoid the broken links.